backend/app/utils.py

```
@@ -122,79 +106,66 @@
 
 
 def verify_reset_token(token: str) -> str:
-    """Verify a reset token and return the email if valid."""
     try:
         payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
-        email: str = payload.get("sub")
+        email = payload.get("sub")
         if email is None:
-            raise HTTPException(status_code=400, detail="Invalid token")
+            raise HTTPException(status.HTTP_400_BAD_REQUEST, detail=INVALID_TOKEN)
         return email
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)